APE18.rst

```diff
@@ -5,12 +5,18 @@ author: Stuart Mumford and Thomas Robitaille
 
 date-created: 2020 October 16
 
-date-last-revised: 2021 February 25
+date-last-revised: 2024 July 12
 
 type: Standard Track
 
 status: Accepted
 
+.. note::
+  NEP 29 has been superseded by the scientific python ecosystem coordination guideline
```

Comment:
```suggestion
  NEP 29 has been superseded by the Scientific Python ecosystem coordination guideline
```

Reply:
I directly copied from NEP 29, where it is all lowercase. In SPEC 0 itself it is uppercase.